modules/docs/arrow-docs/docs/docs/typeclasses/intro/README.md

```diff
@@ -105,6 +105,10 @@ We will list them by their hierarchy.
 
 - [`Bimonad`]({{ '/docs/typeclasses/bimonad/' | relative_url }}) - both monad and comonad
 
+- [`Bifunctor`]({{ '/docs/typeclasses/bifunctor' | relative_url }}) - lifts two functions into a new context
```

Comment:
Think functor, but for two values in the container. The lifting functions to context is more mathy :P

Reply:
I didn't mean write it literally, but I'll take it hahaha